src/core_plugins/console/public/src/directives/help.html

```diff
@@ -1,36 +1,42 @@
-<h3>Help</h3>
+<div class="localDropdownContent localDropdownSection">
```

Comment:
> `localDropdownSection` is typically used as a child of this element but can sometimes be applied to it directly

Is there guidance regarding when this is necessary? I played around with the example in the UI docs and it seems like everything works just fine with or without it. Could this mean that bottom padding in the `localDropdownContent` would be sufficient?